doc/source/user_guide/ug_running.rst

```diff
@@ -137,8 +135,7 @@ The **casescripts/** directory holds scripts used to create the case and can
 largely be ignored.  Once a case is created, the **cice.build** script should be run
 interactively and then the case should be submitted by executing the 
 **cice.submit** script interactively.  The **cice.submit** script
-simply submits the **cice.run script**.  
-You can also submit the **cice.run** script on the command line.
```

Comment:
I would not delete this last line. I often run tests and cases interactively on my workstation.

The rest looks good!

Reply:
That's a reasonable point.  I have add a sentence in this section to indicate the scripts can also be run interactively and submitted manually.